<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Главное меню</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>601</width>
     <height>301</height>
    </rect>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <column>
    <property name="text">
     <string>ФИО</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Пол</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Возраст</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Звание</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Отряд</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Машина</string>
    </property>
   </column>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>300</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Выборка по:</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>300</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>ФИО</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ПОЛ</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ВОЗРАСТ</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ЗВАНИЕ</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ОТРЯД</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>МАШИНА</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>148</x>
     <y>300</y>
     <width>31</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>&gt;</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>&lt;</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>=</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>178</x>
     <y>300</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>289</x>
     <y>299</y>
     <width>313</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Обновить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>360</y>
     <width>291</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Редактировать 
 otryad</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>287</x>
     <y>360</y>
     <width>314</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Редактировать 
 sosttr</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>287</x>
     <y>418</y>
     <width>314</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Редактировать 
 transport</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>418</y>
     <width>291</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Редактировать 
 sotrudniki</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>320</y>
     <width>603</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Печать</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
